### ISSUE TITLE
Change default for "--derive_times" option

### ISSUE THREAD

**Issue by ajcse1** [jaws/jaws#110]
By default JAWS should include this in output file and there should be an option to turn it off

**Comment by ajcse1** [comment]
Done. Option renamed as --no_drv_tm/--no_derive_times to turn it off.
